servers/src/test/java/tachyon/master/journal/JournalFormatterTestBase.java

```diff
@@ -77,6 +79,41 @@
   protected OutputStream mOs;
   protected InputStream mIs;
 
+  //map that holds test journal entries
+  protected Map<JournalEntryType, JournalEntry> mDataSet =
+      ImmutableMap.<JournalEntryType, JournalEntry>builder()
```

Comment:
Also shall we add a test for checking the size of `mDataset` is equal to the size of JournalEntryType's values? That will require the unit test for any further added Journal entry.